zjit/src/hir.rs

```diff
@@ -452,6 +475,8 @@ pub enum Insn {
     GetLocal { level: NonZeroU32, ep_offset: u32 },
     /// Set a local variable in a higher scope
     SetLocal { level: NonZeroU32, ep_offset: u32, val: InsnId },
+    GetSpecialSymbol { symbol_type: SpecialBackrefSymbol, state: InsnId },
+    GetSpecialNumber { svar: u64, state: InsnId },
```

Comment:
```suggestion
    GetSpecialNumber { nth: u64, state: InsnId },
```

This is called `nth` internally that seems like it communicates more intent